packages/sdk-vue/src/types.ts

```diff
@@ -54,10 +54,10 @@ export interface FusionAuthConfig {
   onAutoRefreshFailure?: (error: Error) => void;
 
   /**
-   * Pass in `useCookie` from nuxt/app [useCookie](https://nuxt.com/docs/api/composables/use-cookie).
+   * Pass in `useCookieType` from nuxt/app [useCookie](https://nuxt.com/docs/api/composables/use-cookie).
```

Comment:
This comment shouldn't be changed because the thing they need to pass in is still called `useCookie`